### PR TITLE
Updates the police cap sprite

### PR DESCRIPTION
**Pull request by Bamhalazam**
What it says on the tin. Recolors the police hat sprite to be more in line with the rest of marshals colours (instead of some dorky shade of blue). Also makes the female bowler hat look more like a bowler hat

Both male and female variations. 

image

### CODE REVIEW

**Comment by cdb-is-not-good** [comment]
Neat but couldn't this be an alt sprite?

**Comment by Bamhalazam** [comment]
> Neat but couldn't this be an alt sprite?

the m/f variations already are an alt sprite, and i simply just recolored them. The original ones don't match anything, didn't change these caps' look (save for a few pixels) compared to the original if that's what you're asking. Shouldn't be much of a difference!

**Comment by Bamhalazam** [comment]
> > Neat but couldn't this be an alt sprite?
> 
> the m/f variations already are an alt sprite, and i simply just recolored them. The original ones don't match anything, didn't change these caps' look (save for a few pixels) compared to the original if that's what you're asking. Shouldn't be much of a difference! 

wtf was i thinking when i wrote this

TL;DR no because theres no point to having 4 sprites with minuscule color differences (barely noticeable but enough to be annoying)

**Comment by cdb-is-not-good** [comment]
Meh. Fine, fix the conflict and we'll merge it

**Comment by Trilbyspaceclone** [comment]
Please remake the pr if you wish for it to be merged. As this one has conflicts
